swarm/api/http/server.go

```diff
@@ -732,16 +734,18 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 }
 
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	log.Debug(fmt.Sprintf("HTTP %s request URL: '%s', Host: '%s', Path: '%s', Referer: '%s', Accept: '%s'", r.Method, r.RequestURI, r.URL.Host, r.URL.Path, r.Referer(), r.Header.Get("Accept")))
+	req := &Request{Request: *r, ruid: uuid.New()[:8]}
```

Comment:
`ruid` is now part of the Request Context. In the future it might make sense to propagate it further down the services used by the HTTP server, so that we can track a given request end-to-end.

At the moment we have logs when the request serving starts, and when the request serving ends (when using `Respond`), but for some of the handlers, we have very little tracing.